### ISSUE TITLE
layoutlmv2 visual backbone pretrained on publaynet 

### ISSUE THREAD

**Issue by gauravlochab** [huggingface/transformers#16191]
@NielsRogge Amazing work thanks for sharing,
Does layoutlmv2's visual backbone utilize ResNeXt-FPN trained on Publaynet as described in the paper? If so, could you please point out where the weights are used to initialize the model in the repo?

**Comment by NielsRogge** [comment]
Hi,

The LayoutLMv2 authors indeed leveraged a ResNeXT-FPN for the visual backbone. It was taken from this repo: https://github.com/hpanwar08/detectron2.

However, we've just added the [Document Image Transformer (DiT)](https://huggingface.co/docs/transformers/master/en/model_doc/dit), which actually outperforms the ResNeXT model on PubLayNet with the same object detection framework (Mask R-CNN), reaching an AP of 0.945/0.949 for DiT-base/large respectively.

Take a look at this Space for a demo: https://huggingface.co/spaces/nielsr/dit-document-layout-analysis

**Comment by NielsRogge** [comment]
I don't think Microsoft open-sourced this weights initialization. This was probably part of their pre-training code (which is not open-sourced).

Weights can be found here: https://github.com/hpanwar08/detectron2#benchmarking

**Comment by gauravlochab** [comment]
then how does the visual backbone extracts features from the image? I can see the model config for detectron2 but I can't figure out what weights it is getting initialized with?

**Comment by gauravlochab** [comment]
I want to override the weights for the model but when I pass the weight's path in the visual backbone class the model is not picking up the weights but the MODEL.WEIGHTS is showing the model path string

**Comment by gauravlochab** [comment]
got the weights from the backbone and loaded the custom weights

**Comment by riteshkumarumassedu** [comment]
Hi @gauravlochab,
Could you please share how you managed load the custom weights?
